admin/controllers/results.js

```diff
@@ -57,20 +56,12 @@ controller.getViewResultsPage = async (req, res, next) => {
       breadcrumbs: req.breadcrumbs()
     })
   }
-
-  if (!schoolScoreRecord) {
-    return res.render('availability/admin-window-unavailable', {
-      isBeforeStartDate: checkWindow && currentDate.isBefore(checkWindow.adminStartDate)
-    })
-  }
-  const pupilData = resultPresenter.getResultsViewData(pupils)
-  const nationalScore = resultPresenter.formatScore(checkWindow.score)
-  schoolScore = resultPresenter.formatScore(schoolScoreRecord.score)
+  const pupilWithStatuses = resultService.assignResultStatuses(pupilResultData)
+  const pupilData = resultPresenter.getResultsViewData(pupilWithStatuses)
   return res.render('results/view-results', {
     pupilData,
+    maxMark: config.LINES_PER_CHECK_FORM,
```

Comment:
@jon-shipley you can get a scenario where there's no maxMark simply because all people are marked as not taking the check. That could potentially make this fragile.
The only stable way of receiving that information would be from the check form.
Since we have an env variable that defines the lines of check forms and to the best of my knowledge this hasn't changed it only makes sense to point to that.